planning/behavior_path_planner/src/behavior_tree_manager.cpp

```diff
@@ -158,4 +143,31 @@ void BehaviorTreeManager::resetGrootMonitor()
   }
 }
 
+void BehaviorTreeManager::get_debug_msg_array()
+{
+  scene_module_visitor_ptr_ = std::make_shared<SceneModuleVisitor>();
+  for (const auto & module : scene_modules_) {
+    if (module->name() == "LaneChange") {
+      auto visitor_ptr = std::make_shared<SceneModuleVisitor>();
+      module->accept_visitor(visitor_ptr);
+      const auto get_ptr = visitor_ptr->get_lane_change_debug_msg_array();
+      if (get_ptr) {
+        scene_module_visitor_ptr_->set_lane_change_debug_ptr(get_ptr);
+      }
```

Comment:
Just curious, why do you use this implementation instead of 

```
for (const auto & module : scene_modules_) {
      module->accept_visitor(scene_module_visitor_ptr_);
} 
```
?

Reply:
Yeah. This make senses. Fixed in 57c1ed16af196a0dfcd745d11a0ce6065fccac54